vowpalwabbitslim/opts.cc

```diff
@@ -0,0 +1,78 @@
+#include "opts.h"
+
+#include <cstdlib>
+#include <cctype>
+
+namespace vw_slim {
+	// && used to avoid constant string copying
+	void find_opt(std::string const& command_line_args, std::string arg_name, std::vector<std::string>& out_values)
+	{
+		// append space to search for '--quadratic '
+		arg_name += ' ';
```

Comment:
Any reason not to use <regex> instead of manual parser?

Reply:
additional unavailable dependencies

---
In reply to: [188485450](https://github.com/JohnLangford/vowpal_wabbit/pull/1481#discussion_r188485450) [](ancestors = 188485450)